pwiz_tools/Skyline/TestPerf/DiaUmpireVendorFormatTest.cs

```diff
@@ -148,7 +148,7 @@ protected override void DoTest()
             RunUI(() => SkylineWindow.SaveDocument(documentFile));
 
             // Launch the wizard
-            var importPeptideSearchDlg = ShowDialog<ImportPeptideSearchDlg>(SkylineWindow.ShowImportPeptideSearchDlg);
+            var runPeptideSearchDlg = ShowDialog<ImportPeptideSearchDlg>(SkylineWindow.ShowRunPeptideSearchDlg);

```

Comment:
It would probably have been better if the variable "importPeptideSearchDlg" had not been renamed so that it makes it easier for us to see that this is the only line of code that changed in this file.
(There are some other test files that have the same issue)

Reply:
Done: https://github.com/ProteoWizard/pwiz/pull/3004/commits/ea257cb417f9ca0ac30d147ef4cdbe20c8cc36c9